xbmc/interfaces/python/XBPython.h

```diff
@@ -19,8 +19,6 @@
 #include <memory>
 #include <vector>
 
-#define g_pythonParser CServiceBroker::GetXBPython()
```

Comment:
seems like `#include "ServiceBroker.h"` can be remove in this file.

Reply:
good catch